File: bee2164.py
import math

def binet(n):
    a = math.pow(((1 + math.sqrt(5)) / 2), n)
    b = math.pow(((1 - math.sqrt(5)) / 2), n)
    return (a - b) / math.sqrt(5)

def main():
    n = int(input())
    binet_value = binet(n)
    # Printar com uma casa decimal.
    print(f"{binet_value:.1f}")

main()

File: bee_2164.c
#include <stdio.h>
#include <math.h>

double binet(int n) {
    double a = pow(((1 + sqrt(5)) / 2), n);
    double b = pow(((1 - sqrt(5)) / 2), n);
    return (a - b) / sqrt(5);
}

int main() {
    int n;
    scanf("%d", &n);
    double binet_value = binet(n);
    printf("%.1f\n", binet_value);

    return 0;
}